src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderControlOptionRun.java

```diff
@@ -90,13 +94,15 @@ public DockerBuilderControlOptionRun(
         this.tty = tty;
         this.hostname = hostname;
         this.user = user;
+        this.extraGroupsString = extraGroupsString;
         this.bindPorts = bindPorts;
         this.memoryLimit = memoryLimit;
         this.memorySwap = memorySwap;
         this.cpuShares = cpuShares;
         this.shmSize = shmSize;
         this.bindAllPorts = bindAllPorts;
         this.macAddress = macAddress;
+        this.extraHostsString = extraHostsString;
```

Comment:
This looks like feature creep to me - looks like you're adding support for extraHosts here, and that wasn't part of this PR's description.
We either need to enhance the description or split this change into its own PR.